docs/Guides/getting-startted/working-with-mcap.md

```diff
@@ -0,0 +1,37 @@
+# Working With MCAP
+
+Once you have MCAP data to work with, you may want to **inspect, visualize, and store your MCAP files**.
+
+### CLI commands
+
+[Install the mcap CLI tool](https://github.com/foxglove/mcap/tree/main/go/cli/mcap#installing) to start working with your MCAP files.
+
+To validate an MCAP file:
+
+```
+$ mcap doctor demo.mcap
+```
+
+To report summary statistics on an MCAP file to get information like number of messages, start and end time, channels, etc.:
+
+```
+$ mcap info demo.mcap
```

Comment:
We should have outputs after these commands.

This introduces a bit of a sync issue because it will be tough to keep these guides up to date with changes to the tooling. But I think even slightly outdated outputs would be preferable to no outputs for this kind of doc.